### PR TITLE
Add Indonesian sources

### PR DESCRIPTION
**Pull request by naufaljct48**
Add new Indonesian sources:
- Kiryuu.id
- Komiku.com
- KomikTap (18+)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,9 @@
     "typescript": "^4.3.2"
   },
   "dependencies": {
+    "configstore": "^6.0.0",
```
Comment (TheNetsky) [inline package.json:37]: Why are you adding this?

File: src/Kiryuu/KiryuuParser.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+import { MangaStreamParser } from "../MangaStreamParser";
```
Comment (TheNetsky) [inline src/Kiryuu/KiryuuParser.ts:2]: Reason for custom parser?

File: src/KomikTap/KomikTapParser.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+import { MangaStreamParser } from "../MangaStreamParser";
```
Comment (TheNetsky) [inline src/KomikTap/KomikTapParser.ts:2]: Reason for custom parser?

File: src/Komiku/KomikuParser.ts [hunk 4]
```diff
@@ -0,0 +1,139 @@
+/* eslint-disable linebreak-style */
+import { MangaStreamParser } from "../MangaStreamParser";
```
Comment (TheNetsky) [inline src/Komiku/KomikuParser.ts:2]: ****

File: src/MangaStreamParser.ts [hunk 5]
```diff
@@ -1,404 +1,564 @@
 /* eslint-disable linebreak-style */
 import {
```
Comment (TheNetsky) [inline src/MangaStreamParser.ts:2]: Don't reformat, thanks.

File: src/Komiku/Komiku.ts [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+} from "paperback-extensions-common";
+
+import { MangaStream, getExportVersion } from "../MangaStream";
+
+import { KomikuParser } from "./KomikuParser";
+
+const KOMIKU_DOMAIN = "https://komiku.com";
+
+export const KomikuInfo: SourceInfo = {
+  version: getExportVersion("0.0.6"),
```
Comment (TheNetsky) [inline src/Komiku/Komiku.ts:16]: Why start with 0.0.6?

File: src/KomikTap/KomikTap.ts [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+} from "paperback-extensions-common";
+
+import { MangaStream, getExportVersion } from "../MangaStream";
+
+import { KomikTapParser } from "./KomikTapParser";
+
+const KOMIKTAP_DOMAIN = "https://194.233.66.232";
+
+export const KomikTapInfo: SourceInfo = {
+  version: getExportVersion("0.0.6"),
```
Comment (TheNetsky) [inline src/KomikTap/KomikTap.ts:16]: Why start with 0.0.6?

File: src/KomikTap/KomikTap.ts [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+  SourceInfo,
+  TagType,
+  ContentRating,
+} from "paperback-extensions-common";
+
+import { MangaStream, getExportVersion } from "../MangaStream";
+
+import { KomikTapParser } from "./KomikTapParser";
+
+const KOMIKTAP_DOMAIN = "https://194.233.66.232";
```
Comment (TheNetsky) [inline src/KomikTap/KomikTap.ts:13]: Use valid domain.

**Review by TheNetsky — COMMENTED** [review]
Are you sure you're using the correct eslint?

Please look over your code entirely.

File: package.json (hunk 1 above)
Comment (naufaljct48) [reply package.json:37]: idk, there's some error when i try to "paperback serve"
but i think it's no need

**Comment by naufaljct48** [comment]
> Are you sure you're using the correct eslint?
> 
> Please look over your code entirely.

I'm just doing "npm i"
npm i -g paperback-cli@2.0.0-alpha.13
npm install -D @types/cheerio@0.22.21

File: src/Kiryuu/KiryuuParser.ts (hunk 2 above)
Comment (naufaljct48) [reply src/Kiryuu/KiryuuParser.ts:2]: As you can see, the "Author", "Artists" things won't show. Then custom parser are needed.
Also i need to trim the Titles.

**Comment by TheNetsky** [comment]
> > Are you sure you're using the correct eslint?
> > Please look over your code entirely.
> 
> I'm just doing "npm i" npm i -g paperback-cli@2.0.0-alpha.13 npm install -D @types/cheerio@0.22.21

You need to have ESlint installed in your VSCode.

File: src/KomikTap/KomikTapParser.ts (hunk 3 above)
Comment (naufaljct48) [reply src/KomikTap/KomikTapParser.ts:2]: As you can see, the "Author", "Artists" things won't show. Then custom parser are needed.

File: src/Komiku/KomikuParser.ts (hunk 4 above)
Comment (naufaljct48) [reply src/Komiku/KomikuParser.ts:2]: As you can see, the "Author", "Artists" things won't show. Then custom parser are needed.
Also i need to trim the Titles.

File: src/MangaStreamParser.ts (hunk 5 above)
Comment (naufaljct48) [reply src/MangaStreamParser.ts:2]: I'm sorry about that

File: src/KomikTap/KomikTap.ts (hunk 8 above)
Comment (naufaljct48) [reply src/KomikTap/KomikTap.ts:13]: Unfortunately, i have tried with proper domain.
But it doesn't work when i test...

File: src/Komiku/Komiku.ts (hunk 6 above)
Comment (naufaljct48) [reply src/Komiku/Komiku.ts:16]: Oh, i thought it was 0.0.6 version lol my bad

**Comment by naufaljct48** [comment]
> > > Are you sure you're using the correct eslint?
> > > Please look over your code entirely.
> > 
> > 
> > I'm just doing "npm i" npm i -g paperback-cli@2.0.0-alpha.13 npm install -D @types/cheerio@0.22.21
> 
> You need to have ESlint installed in your VSCode.

I installed it already, can you pm me at discord? i can't dm you kurotenshi#6969

**Comment by TheNetsky** [comment]
Join Discord, enter dev channel.

**Comment by naufaljct48** [comment]
> Join Discord, enter dev channel.

Joined already, but i don't see any dev channel

**Comment by TheNetsky** [comment]
> > Join Discord, enter dev channel.
> 
> Joined already, but i don't see any dev channel

I'll wait for you to find it then.

**Pull request closed without merging**